content/kubermatic/main/tutorials-howtos/audit-logging/_index.en.md

```diff
@@ -127,3 +127,52 @@ For exiting clusters, you can go to the cluster page, edit your cluster and enab
 KKP also supports enabling Audit Logging on the datacenter level. In this case, the option is enforced on all user-clusters in the datacenter. The user-cluster level flag is ignored in this case.
 
 To enable this, you will need to edit your [datacenter definitions in a Seed]({{< ref "../../tutorials-howtos/administration/dynamic-data-centers/" >}}), and set `enforceAuditLogging` to `true` in the datacenter spec.
+
+## Webhook Backend For Audit Logs
+
+User clusters can be also be configured to send audit logs to a [webhook backend](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#webhook-backend), KKP admin needs to create kubernetes secret that holds the audit webhook backend configuration, on the seed cluster which can then be used to enable webhook backend at the cluster or the datacenter level.
+
```

Comment:
Can you also add an example of how the secret should look like?

Reply:
Added.